ingestion/src/metadata/orm_profiler/metrics/core.py

```diff
@@ -44,9 +45,13 @@ def _label(_fn):
     @wraps(_fn)
     def inner(self, *args, **kwargs):
         res = _fn(self, *args, **kwargs)
-
         # If the metric computation returns some value
         if res is not None:
+            try:
+                if pd.isnull(res):
+                    res = None
+            except ValueError:
+                pass
```

Comment:
Can you give more detaisl?

Reply:
so the pandas doinging computing on list full of None returns NaN, cannot be handled any other way

Reply:
if we don't handle it here, API gives us 404 as it goes as `"mean": NaN`
and NaN == None is False, so had to handle it like this.